defradb_versioned_docs/version-1.0/guides/deployment.md

```
@@ -0,0 +1,127 @@
+4. Build a specific playground version of DefraDB. Use the go flags environment variable, instructing the compiler to include the playground directly embedded in all files. Execute the [go binary embed](https://pkg.go.dev/embed) command, producing a binary of approximately 4MB.
+
+### Docker Deployments
+
+Docker deployments are designed for containerized environments. The main prerequisite is that Docker should be installed on your machine.
+
+The steps for Docker deployment are as follows:
+
+1. Install Docker by referring to the [official Docker documentation](https://docs.docker.com/get-docker/).
+2. Navigate to the root of the repository where the Dockerfile is located.
+3. Run the following command: 
+`docker build -t defra -f tools/defradb.containerfile `
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)